public/app/features/dashboard/components/SaveDashboard/forms/SaveDashboardForm.tsx

```diff
@@ -61,7 +61,7 @@ export const SaveDashboardForm = ({
         <Stack direction="column" gap={2}>
           {hasTimeChanged && (
             <Checkbox
-              checked={options.saveTimerange}
+              checked={!!options.saveTimerange}
```

Comment:
```suggestion
              checked={Boolean(options.saveTimerange)}
```
maybe?

Reply:
That works too and could be argued to be a bit clearer, but since we're already using `!!` in this file, let's keep it for consistency.